src/main.rs

```diff
@@ -1130,14 +1117,15 @@ fn cmd_certify(out: &mut dyn Write, cfg: &Config, sub_args: &CertifyArgs) -> Res
     let eula = if let Some(eula) = eula_for_criteria(&store.audits, &criteria) {
         eula
     } else {
-        error!("couldn't get description of criteria");
-        panic_any(ExitPanic(-1));
+        Err(eyre!("couldn't get description of criteria"))?
```

Comment:
I'd reword this to describe the state rather than the activity that failed, though I don't have enough domain knowledge to know what wording to suggest instead. Something like "there is no description of the criteria available"

Reply:
Hmm so the purpose of the print-and-exit pattern is to not include any of the more "debug-like" features of Report like the backtrace (while still allowing it to exist for things like i/o errors).

In https://github.com/bholley/cargo-vet/pull/140/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR1190

I just changed the pattern to `writeln!(out` while still doing a PanicExit. This would make them more debug-y right? Are you planning to mess with the report format too?

Reply:
> Hmm so the purpose of the print-and-exit pattern is to not include any of the more "debug-like" features of Report like the backtrace (while still allowing it to exist for things like i/o errors).

my plan for that was to customize the error reporting to not include that extra information so we can keep the representation of errors consistent.

> I just changed the pattern to `writeln!(out` while still doing a PanicExit. This would make them more debug-y right? Are you planning to mess with the report format too?

No plans to mess with the report format, my expectation is that any errors that get emitted as part of the report will immediately be discarded and will be effectively represented further in the process lifecycle by the "report contained errors" error that gets emitted at the end.

Reply:
Hmm I'm not sure I understand the pitch here. It feels like there must be some kind of distinct error type to distinguish between user-designed errors and "errors we didn't explicitly handle and are exposed more raw"?